src/entrypoints/options/pages/general/index.tsx

```diff
@@ -1,12 +1,75 @@
+import { toast } from 'sonner'
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from '@/components/ui/alert-dialog'
+import { Button } from '@/components/ui/button'
+import { ConfigCard } from '../../components/config-card'
 import { PageLayout } from '../../components/page-layout'
 import { ReadConfig } from './read-config'
 import TranslationConfig from './translation-config'
 
+function resetToDefaultConfig() {
```

Comment:
this is not a right way to do. you should do something like

```tsx
  const setConfig = useSetAtom(writeConfigAtom);
  function resetToDefaultConfig() {
    setConfig(DEFAULT_CONFIG);
  }
  ```
  
  to understand why, you can check :https://github.com/mengxi-ream/read-frog/blob/054a7674283c6767f57eddbcd85ebaf382372e07/src/utils/atoms/config.ts